src/axom/spin/internal/linear_bvh/build_radix_tree.hpp

```diff
@@ -264,7 +264,7 @@ void sort_mcodes(Array<uint32>& mcodes, int32 size, const ArrayView<int32> iter)
 
     std::stable_sort(iter.begin(),
                      iter.begin() + size,
-                     [=](int32 i1, int32 i2) { return mcodes[i1] < mcodes[i2]; });
+                     [&](int32 i1, int32 i2) { return mcodes[i1] < mcodes[i2]; });
 
   );
```

Comment:
At the axom meeting on 1/28/2023, we discussed an alternate resolution of converting the `mcodes` parameter to `sort_mcodes` from an `axom::Array` to an `axom::ArrayView` which would avoid the copy.  

Should I apply this suggestion instead of the `by-reference` capture?

Reply:
Let's put in the by-reference change at this point, and later change to use an `ArrayView`.